### DISCUSSION TITLE
Include Pull Request Description in the Merge Commit

### DISCUSSION THREAD

**Discussion by amjltc295** [community/community#4523] [Pull Requests]
_Note: the same issue has been mentioned [here](https://github.com/isaacs/github/issues/467) but that repo seems deprecated._

Can we include pull request description in the merge commit?

For example:
```
Merge pull request #N from repo/branch.
<PR-title>  # This is what we have now
<PR-description>  # Can we add this?  
```

**Comment by willsmythe** [comment]
The default commit message presented when merging is now configurable, and there's an option to use the PR title and description by default: https://github.blog/changelog/2022-08-23-new-options-for-controlling-the-default-commit-message-when-merging-a-pull-request/
